test/run-k8s-integration.sh

```diff
@@ -26,19 +26,25 @@ readonly gke_release_channel=${GKE_RELEASE_CHANNEL:-""}
 readonly gke_node_version=${GKE_NODE_VERSION:-}
 readonly gce_region=${GCE_CLUSTER_REGION:-}
 readonly storageclass_files=${STORAGECLASS_FILES:-}
-readonly use_staging_driver=${USE_STAGING_DRIVER:-false}
+readonly use_gke_driver=${USE_STAGING_DRIVER:-false}
```

Comment:
we should eventually change USE_STAGING_DRIVER too.

Reply:
ack.